README.markdown

```diff
@@ -41,6 +41,10 @@ To achieve git-hubflow completion nirvana:
                 esac
         }
 
+ 3. If you are using [git-bash-completion](https://github.com/markgandolfo/git-bash-completion) you may consider change your PS1 to also show differences between the current branch and the base branch: PS1='[\u@\h \W$(__git_ps1_diff_base " %s")$(__git_ps1 " (%s)")]\$ '
```

Comment:
This documentation isn't clear.
* PS1 isn't a universally known "term" (I asked my devs, they didn't know)
* It wasn't clear whether this should be copied wholesale without edit into .bash_profile, or whether it needed exporting
* It really should be in a code block, so it can be copied out directly.